### DISCUSSION TITLE
Explanation on Tutorial frontend

### DISCUSSION THREAD

**Discussion by spiralcb** [reflex-dev/reflex#1836] [Q&A]
Hello, 

I think it's a python syntax but I don't understand the following : 

return rx.box(
        *[
            qa(question, answer)
            for question, answer in qa_pairs
        ]

What does the *[....] syntax ?

Thanks in advance

**Comment by Lendemor** [comment]
Hello.
This is "list unpacking".

It take a list (here `[ qa(question, answer) for question, answer in qa_pairs]` ) and pass each element of said list as arguments of `rx.box(...)`

Here is some reference you can read about it : https://www.pythontutorial.net/python-basics/python-unpack-list/

**Comment by spiralcb** [comment]
Hello, 

Thanks a lot.
